docs/advanced-usage.rst

```diff
@@ -538,6 +538,37 @@ Here's an example using brotli encoding via the ``Accept-Encoding`` header:
         headers={"Accept-Encoding": "br"}
     )
 
+Zstandard Encoding
+------------------
+
+`Zstandard <https://datatracker.ietf.org/doc/html/rfc8878>`_
+is a compression algorithm created by Facebook with better compression
+than brotli, gzip and deflate (see `benchmarks <https://facebook.github.io/zstd/#benchmarks>`_)
+and is supported by urllib3 if the `zstandard <https://pypi.org/project/zstandard/>`_ package is installed.
```

Comment:
Sphinx doesn't like having two links with the same text, maybe we include the word "package" within the hyperlink text here?